src/components/Header/Header.component.tsx

```diff
@@ -14,48 +13,31 @@ import type { Themes } from '@/types/themes.type.ts';
 import styles from './Header.module.css';
 
 interface HeaderProps {
-    activePage: Pages;
     totalProductsInCart: number;
     handleActivePage: (page: Pages) => void;
     theme: Themes;
     setTheme: (theme: Themes) => void;
 }
 
-export const Header: React.FC<HeaderProps> = ({ totalProductsInCart, handleActivePage, activePage, theme, setTheme }) => (
+export const Header: React.FC<HeaderProps> = ({ totalProductsInCart, handleActivePage, theme, setTheme }) => (
     <div className={styles.headerWrapper}>
         <div className={styles.header}>
             <div className={styles.leftIcons}>
                 <IconWithRedirect icon={images.maCrownIcon} url={links.home} alt={'masters academy icon'}></IconWithRedirect>
```

Comment:
I hope that this link should lead to "Masters Academy" website, but it's up to you. Later you may add your logo there…